src/templates/CentralTemplateProvider.ts

```diff
@@ -258,12 +259,24 @@ export class CentralTemplateProvider implements Disposable {
     }
 
     private async getLatestTemplates(context: IActionContext, provider: TemplateProviderBase, latestTemplateVersion: string): Promise<ITemplates | undefined> {
-        if (!this.templateSource || this.templateSource === TemplateSource.Latest || this.templateSource === TemplateSource.Staging) {
-            context.telemetry.properties.templateSource = 'latest';
-            const result: ITemplates = await provider.getLatestTemplates(context, latestTemplateVersion);
-            await provider.cacheTemplateMetadata(latestTemplateVersion);
-            await provider.cacheTemplates(context);
-            return result;
+        try {
+            if (!this.templateSource || this.templateSource === TemplateSource.Latest || this.templateSource === TemplateSource.Staging) {
+                context.telemetry.properties.templateSource = 'latest';
+                const result: ITemplates = await provider.getLatestTemplates(context, latestTemplateVersion);
+                await provider.cacheTemplateMetadata(latestTemplateVersion);
+                await provider.cacheTemplates(context);
+                return result;
+            }
+        } catch (error) {
+            // If we failed to get templates for a non-English language, try English
+            const language = provider.resourcesLanguage || getScriptResourcesLanguage();
+            if (language !== 'en-US') {
```

Comment:
We should import and use this constant for this string since it's already declared.

https://github.com/microsoft/vscode-azurefunctions/blob/4d43353ad99c360a0ad09cd980b31986fe8e00d2/src/templates/script/getScriptResourcesLanguage.ts#L8